### PR TITLE
AML check update

### PR DESCRIPTION
**Pull request by DemolaJames**
# Task:

## Description

- Updated the api to return true for case where we do not get a 200 response for check provider 
- Added sentry logging for the checks that fall in the category above

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## FE Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

## User Facing Checklist

- [ ] I fully understand the user problem this PR is solving
- [ ] I know who the target user is for this PR and have a deep understanding of that user
- [ ] I have tried this flow thinking from the pov of the target user for this PR
- [ ] I (or working w someone on team) have scheduled a user test for this PR (if it is a large change)

### CODE REVIEW

File: packages/frontend/pages/api/isValidAddress.ts [hunk 1]
```diff
@@ -18,8 +20,16 @@ const handleRequest = async (req: NextApiRequest, res: NextApiResponse) => {
     )
 
     res.status(200).json({ valid: (data?.[0]?.rating ?? 'highRisk') !== 'highRisk' })
```
Comment (akwritescode) [inline packages/frontend/pages/api/isValidAddress.ts:22]: what will happen, if you get a 404 to to the call in line 10?

Comment (DemolaJames) [reply packages/frontend/pages/api/isValidAddress.ts:22]: All errors not 200 goes Into the catch block...I tested for 400, 403, 404.

**Review by akwritescode — APPROVED** [review]
lgtm

File: packages/frontend/pages/api/isValidAddress.ts (hunk 1 above)
Comment (KMKoushik) [inline packages/frontend/pages/api/isValidAddress.ts:22]: Did you try with risky address? To my knowledge, if you don't return at this point code will continue and always return as true

That brings my 2nd point, Can we add tests to make sure this always works? Can mock the `chainalysis` requests for different cases

**Review by KMKoushik — CHANGES_REQUESTED** [review]

**Review by KMKoushik — APPROVED** [review]

**Pull request merged**